test/arbitrum/JPEGzMainnetOracle.t.sol

```diff
@@ -0,0 +1,505 @@
+      orchestrator,
+      divisor,
+      ratio,
+      burnFee,
+      _mintFee,
+      liquidationPenalty,
+      address(jpegzOracle),
+      jpegz,
+      address(weth),
+      address(ethOracle),
+      address(ethOracle),
+      treasury,
+      0
+    );
+  }
+
+  function testConstructor_ShouldRevert_WhenLiquidationPenaltyIsHigh(
+    uint256 _liquidationPenalty,
+    uint256 _ratio
+  ) public {
+    if (_liquidationPenalty + 100 < _liquidationPenalty) {
+      return;
+    }
+
+    if ((_liquidationPenalty + 100) >= _ratio) {
+      vm.expectRevert(
+        "VaultHandler::constructor: liquidation penalty too high"
+      );
+    }
+
+    ETHVaultHandler vault = new ETHVaultHandler(
+      orchestrator,
+      divisor,
+      _ratio,
+      burnFee,
+      mintFee,
+      _liquidationPenalty,
+      address(jpegzOracle),
+      jpegz,
+      address(weth),
+      address(ethOracle),
+      address(ethOracle),
+      treasury,
+      0
+    );
+
+    if (!((_liquidationPenalty + 100) >= _ratio)) {
+      assertEq(_liquidationPenalty, vault.liquidationPenalty());
+    }
+  }
+
+  function testSetLiquidationPenalty_ShouldUpdateValue(
+    uint256 _liquidationPenalty
+  ) public {
+    if (_liquidationPenalty + 100 < _liquidationPenalty) {
+      return;
+    }
+
+    if ((_liquidationPenalty + 100) >= ratio) {
+      vm.expectRevert(
+        "VaultHandler::setLiquidationPenalty: liquidation penalty too high"
+      );
+    }
+
+    orchestrator.setLiquidationPenalty(ethVault, _liquidationPenalty);
+  }
+
+  function testShouldUpdateRatio(uint256 _ratio) public {
+    if (_ratio < 100) {
+      vm.expectRevert("VaultHandler::setRatio: ratio lower than MIN_RATIO");
+    } else {
+      if (ethVault.liquidationPenalty() + 100 >= _ratio) {
+        vm.expectRevert("VaultHandler::setRatio: liquidation penalty too high");
+      }
+    }
+    orchestrator.setRatio(ethVault, _ratio);
+  }
+
+  function testSetMinimumTCAP_ShouldUpdateValue(uint256 _minimumTCAP) public {
+    vm.expectRevert("Ownable: caller is not the owner");
+    ethVault.setMinimumTCAP(_minimumTCAP);
+
+    vm.expectEmit(true, true, true, true);
+    emit NewMinimumTCAP(address(orchestrator), _minimumTCAP);
+    orchestrator.executeTransaction(
+      address(ethVault),
+      0,
+      "setMinimumTCAP(uint256)",
+      abi.encode(_minimumTCAP)
+    );
+    assertEq(ethVault.minimumTCAP(), _minimumTCAP);
+  }
+
+  function testMint_ShouldCreateTCAP_WhenFeeIsPaid() public {
+    uint256 fee = ethVault.getMintFee(1 ether);
+    vm.startPrank(user);
+    vm.deal(user, 100 ether);
+    ethVault.createVault();
+    ethVault.addCollateralETH{value: 10 ether}();
+    ethVault.mint{value: fee}(1 ether);
+
+    (uint256 id, uint256 collateral, uint256 debt, address owner) = ethVault
+      .vaults(1);
+    assertEq(id, 1);
+    assertEq(collateral, 10 ether);
+    assertEq(debt, 1 ether);
+    assertEq(owner, user);
+    assertEq(address(treasury).balance, fee);
```

Comment:
Just to be sure that our calculations with fees are correct, can you please check that this actually 1.5% of 1 ether?
i.e 
```solidity
unit256 _fee = ((15/10) * 1 ether * JPEGZ_PRICE())/(100 * ETH_PRICE());
assertEq(address(treasury).balance, _fee);
```
using 15/10 as we know that is equal 1.5. I just want to make sure that we've set `uint256 mintFee = 150;` correctly above.

Reply:
added these lines:

```
uint256 _fee = ((15 / 10) * 1 ether * ethVault.getOraclePrice(jpegzOracle)) / (ethVault.oracleDigits() * 10 * ethVault.getOraclePrice(ethOracle));
assertEq(address(treasury).balance, _fee);

```